zombies.js

```diff
@@ -158,7 +211,21 @@
  * @name equip
  * @param {Weapon} itemToEquip  The weapon item to equip.
  */
-
+////////////////////STUDY THIS///////////////////////////
+equip(weapon) {
+  let itemIndex = this._pack.indexOf(weapon);
+  let equipped = this.equipped;
+  let _pack = this._pack;
+  if (!equipped && _pack.includes(weapon) && weapon instanceof Weapon) {
```

Comment:
since you already did an indexOf operation you should use that instead of using includes.
```js
itemIndex > -1 // instead of _pack.includes(weapon)